Source/FakeItEasy.Tests/Configuration/ArgumentCollectionTests.cs

```diff
@@ -98,22 +100,24 @@ public void GetEnumerator_returns_enumerable_that_enumerates_the_arguments_of_th
         {
             var arguments = this.CreateFakeArgumentList(1, 2, 3);
 
-            Assert.That(arguments, Is.EquivalentTo(new[] { 1, 2, 3 }));
+            arguments.ShouldBeEquivalentTo(new[] { 1, 2, 3 });
         }
 
         [Test]
-        public void Should_be_serializable()
+        public void Should_be_serializable_when_arguments_are_equatable()
         {
             // Arrange
-            var collection = new ArgumentCollection(new object[] { new object() }, typeof(object).GetMethod("Equals", new[] { typeof(object) }));
+            object collection = new ArgumentCollection(
+                new object[] { "first argument" },
+                typeof(string).GetMethod("Equals", new[] { typeof(string) }));

```

Comment:
* var → object to so FluentAssertions will let me `BeBinarySerializable`.
* changed the type of the argument because FluentAssertions is stricter than NUnit when checking binary serializability—it compares the objects after deserialization, and a `new object()` would not compare properly